beacon_node/beacon_processor/src/lib.rs

```diff
@@ -498,6 +500,12 @@ impl<E: EthSpec> From<ReadyWork> for WorkEvent<E> {
                 drop_during_sync: false,
                 work: Work::ChainSegmentBackfill(process_fn),
             },
+            ReadyWork::ColumnReconstruction(QueuedColumnReconstruction { process_fn, .. }) => {
+                Self {
+                    drop_during_sync: true,
```

Comment:
Note that we will never perform reconstruction during sync (which is also an upcoming feature), but i think we can think about it a bit more.